src/core/lombok/javac/JavacAST.java

```diff
@@ -174,7 +174,14 @@ private static URI tryGetSbtFile_(JavaFileObject sourcefile) throws Exception {
 		if (sbtMappedVirtualFileRootsField == null) return null;
 		
 		String encodedPath = (String) sbtMappedVirtualFilePathField.get(mappedVirtualFile);
-		if (!encodedPath.startsWith("${")) return null;
+		if (!encodedPath.startsWith("${")) {
+			File maybeAbsoluteFile = new File(encodedPath);
+			if (maybeAbsoluteFile.exists()) {
```

Comment:
Not sure if the check for existence is really necessary, or if we just should assume it's the absolute path and just `return new File(encodedPath);`